Source/ARTConnection+Private.h

```diff
@@ -7,12 +7,20 @@ NS_ASSUME_NONNULL_BEGIN
 
 @class ARTRealtimeInternal;
 
+@interface ARTConnectionRecoveryKey : NSObject
```

Comment:
It would make the code easier to reason about for instances of this class to be immutable. Involving:

1. Adding `readonly` to all properties
2. Adding an init method taking all property values in

Reply:
e062db7a74198d9d0e24b86f2324aa221fd0156e